### PR TITLE
feat: add og image

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Add the meta image for `og:image` and `twitter:image`

Samples of the generated `og:image` (i18n supported):

_concepts_sign-in-experience
_zh-CN_user-management_personal-access-token

### CODE REVIEW

File: docusaurus.config.ts [hunk 1]
```diff
@@ -18,6 +19,15 @@ const localePath = currentLocale === defaultLocale ? '' : currentLocale;
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
+const cfPagesBranch = process.env.CF_PAGES_BRANCH;
+
+console.log('#### Deploying branch ####', cfPagesBranch);
```
Comment (gao-sun) [inline docusaurus.config.ts:24]: do we still needs this?

File: plugins/og-image-generator/fonts/Inter_18pt-Regular.ttf
Comment (gao-sun) [inline plugins/og-image-generator/fonts/Inter_18pt-Regular.ttf:1]: can we use google fonts url to load fonts?

File: plugins/og-image-generator/fonts/index.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * This metadata includes the font name, filename, and weight.
+ * The `name` will be used in the `font-family` CSS property as font faces.
+ *
+ * Also, satori does not support variable fonts, so static fonts must be provided separately
+ * for each weight.
+ *
+ * Note: The `NotoSans` font is used for CJK characters.
+ */
+const fontsMetadata = [
+  { name: 'Inter', filename: 'Inter_18pt-Regular.ttf', weight: 400 },
```
Comment (gao-sun) [inline plugins/og-image-generator/fonts/index.ts:22]: image

this is on their readme

File: docusaurus.config.ts (hunk 1 above)
Comment (charIeszhao) [reply docusaurus.config.ts:24]: Yes, I need to see the console output in prod env (after deployment)

File: plugins/og-image-generator/fonts/Inter_18pt-Regular.ttf
Comment (charIeszhao) [reply plugins/og-image-generator/fonts/Inter_18pt-Regular.ttf:1]: You mean download first and then load them as Buffer?

File: plugins/og-image-generator/fonts/index.ts (hunk 2 above)
Comment (charIeszhao) [reply plugins/og-image-generator/fonts/index.ts:22]: Ah, I see. You are actually suggesting using `fetch` to load Google fonts from URL.

File: docusaurus.config.ts (hunk 1 above)
Comment (gao-sun) [reply docusaurus.config.ts:24]: can we limit this to CF only?

File: plugins/og-image-generator/fonts/Inter_18pt-Regular.ttf
Comment (gao-sun) [reply plugins/og-image-generator/fonts/Inter_18pt-Regular.ttf:1]: no need per offline discussion

**Review by gao-sun — APPROVED** [review]

File: plugins/og-image-generator/fonts/NotoSansJP-Bold.ttf
[the remark this answers is not in the document]
Comment (charIeszhao) [reply plugins/og-image-generator/fonts/NotoSansJP-Bold.ttf:1]: Removed

File: docusaurus.config.ts (hunk 1 above)
Comment (charIeszhao) [reply docusaurus.config.ts:24]: Offline discussed and we move on. This console log will be removed later

**Pull request merged**